contracts/utils/OevSearcherMulticall.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import "./interfaces/IOevSearcherMulticall.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/// @title Contract that enables batched calls to external contracts with value
+/// @notice This contract can be used for two use-cases: (1) In its current
+/// state, it can be used to batch static calls to contracts that do not care
+/// about who the sender is, (2) after extending it, to interact with trusted
+/// contracts (see below for details). It implements a way of batching with value,
+/// and requires none of the calls to revert.
+/// @dev As mentioned above, this contract can be used to interact with trusted
+/// contracts. Such interactions can leave this contract in a privileged
+/// position (e.g., OevSearcherMulticall may be left with a non-zero balance of an
+/// ERC20 token as a result of a transaction sent to it), which can be abused
+/// by an attacker afterwards. In addition, attackers can frontrun interactions
+/// to have the following interaction result in an unintended outcome. A
+/// general solution to these attacks is overriding the externalMulticallWithValue function
```

Comment:
As a note for the future, [we limit the width to 80 characters](https://github.com/api3dao/airnode-protocol-v1/blob/main/.prettierrc.js#L18) but there is no nice way of doing this automatically for NatSpec lines so I format these manually. This line here is too long.